script.md

```diff
@@ -127,54 +127,72 @@ Before we take a look at the rest of the file, we're going to add one more line
+Before moving onto the next part of the config it would be good if we address a few things first. Every time you start the Monero daemon it starts several processes which use different ports to run. The first of these is the P2P service. This is how your node communicates with the rest of the network and keeps itself up to date. Currently the IP address is bound to **0.0.0.0**, this is the best option if you haven't got any kind of custom networking. 
 
 The port number is bound to the default and recommended one. You can of course change this to what ever you like, but be mindful that a wide variety of ports are used by other applications/services, so it's a good idea to stick to the recommended ones. 
 
 Being able to change this port number is great if you cannot forward a certain port on your router, or if your VPN service won't let your specify this one, however, we're going to leave this port as default for now, so take note of it. For our node to be a fully fledged member of the Monero network we need to forward this port on both our firewall and router so that our node can relay its blockchain data.
 
-The other process on the list is the Monero RPC. RPC stands for remote procedure call and is the method used for communication between wallets and nodes. We're not going to go into too much detail in this video, but it is possible for you to allow external connections. Running an RPC service is certainly helpful to those who can't or won't run their own node, but it exposes an entirely different part of the Monero codebase to the Internet. As many of you are will be setting this up on your own personal computer, we advise against this for now.
+The other process on the list is the Monero RPC. RPC stands for remote procedure call and is the method used for communication between wallets and nodes. We're not going to go into too much detail in this video, but it is possible for you to allow external connections. Running an RPC service is certainly helpful for those who don't run their own node, but it exposes an entirely different part of the Monero codebase to the internet. As many of you will be setting this up on your own personal computer, we advise against this for now.
 
-To save time in this video we're going to skip over the next two sections. If you're interested in what they do, please check out the monerodocs for more info.
+__To save time in this video we're going to skip over the next two sections. If you're interested in what they do, please check out the monerodocs for more info.__ (???)
```

Comment:
*skip over the next two sections of this example file. more info - [MoneroDocs](https://monerodocs.org/interacting/monerod-reference)